core/dispatcher/src/main/scala/whisk/core/container/ContainerPool.scala

```diff
@@ -102,7 +102,7 @@ class ContainerPool(
     def maxActive_=(value: Int): Unit = _maxActive = Math.max(0, value)
 
     def resetMaxIdle() = _maxIdle = defaultMaxIdle
-    def resetMaxActive() = _maxActive = defaultMaxActive
+    def resetMaxActive() = _maxActive = ContainerPool.defaultMaxActive
```

Comment:
moved to singleton to it is accessible in invoker (to control depth of pipeline).